### DISCUSSION TITLE
New Demo: ArchiMate® Enterprise Architecture Modeling Language

### DISCUSSION THREAD

**Discussion by kumilingus** [clientIO/joint#2402] [News]
[ArchiMate](https://www.opengroup.org/archimate-forum/archimate-overview) is an **enterprise architecture modeling language** that provides a framework for describing, analyzing, and visualizing the relationships among business domains in an enterprise.

Today's example shows only a subset of the _ArchiMate_ language. Specifically, it is the **Business Layer**, which is one of the three layers of the language. It describes the organizational structure of an enterprise at the level of business units and their internal collaborations.

The **context menu** plays an important role in this example. It allows you to **select the type of relationship immediately when creating a connection** or inserting one element into another. Another notable features of the demo are the switching between different element representations (visuals) and the existence of **implicit and explicit links** (links that only appear when the connected elements are not nested within each other). For more information, see the [ArchiMate Demo](https://codepen.io/jointjs/full/poGpMjY).

<img width="595" alt="image" src="https://github.com/clientIO/joint/assets/3967880/087390db-14c7-4ed9-aaa5-5e0e50288d59">

**Comment by sorenpommer** [comment]
Thanks for this demo. We're interested in drawing with ArchiMate shapes. However, we only see a few sample shapes. Do you plan to make all shapes available? e.g. like these: https://drawio-app.com/blog/new-archimate-3-0-shapes/

**Reply by kumilingus** [reply]
We have no plans to implement the rest of the specification at this moment.
Note that you can implement it at your end as custom shapes.
